build/tasks/build.js

```diff
@@ -81,7 +84,7 @@ module.exports = function( grunt ) {
 
 			// Remove define wrappers, closure ends, and empty declarations
 			contents = contents
-				.replace( /define\([^{]*?{/, "" )
+				.replace( /define\([^{]*?{\s*(?:"use strict";)?/, "" )
```

Comment:
This is for the purposes of transforming our code only so e.g. catching `'use strict'` is unnecessary. Also, why strip comments?